src/lib/template_style/_form.scss

```diff
@@ -19,6 +19,14 @@
   margin-top: 0px;
   margin-bottom: 0px;
 }
+// PORTALS-2053: if checkbox/radio was generated by rjsf, then we do need some space between the elements
+.rjsf {
+  .checkbox,
+  .radio {
+    margin-bottom: 10px;
```

Comment:
Ok for checkboxes and radios to generally have 0 margin top and bottom, but in the context of a rjsf generated form (like in ModalDownload) we need to add some space.